src/openrct2-ui/UiContext.Win32.cpp

```diff
@@ -38,7 +38,7 @@
 static std::wstring SHGetPathFromIDListLongPath(LPCITEMIDLIST pidl)
 {
     std::wstring pszPath(MAX_PATH, 0);
-    while (!SHGetPathFromIDListEx(pidl, &pszPath[0], (DWORD)pszPath.size(), 0))
+    while (!SHGetPathFromIDListW(pidl, &pszPath[0]))
```

Comment:
I'm assuming this changes it to a more backward compatible call (?)

Reply:
Yes, the extended version only makes sense if you use the non-standard parameters. In our case, it was using the standard ones